ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -1158,11 +1158,13 @@ export default class ConfirmTransactionBase extends Component {
     } = this.getNavigateTxData();
 
     let functionType;
+    let isContractInteractionFromDapp = false;
```

Comment:
Can we make this a `const` and set its value to:

```
const isContractInteractionFromDapp = txData.type === TRANSACTION_TYPES.CONTRACT_INTERACTION && txData.origin !== 'metamask';
```

...and use that for the `if`?

Reply:
done in 358dacb0f5